packages/amplify-category-auth/src/utils/transform-user-pool-group.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const fs = require('fs');
+const { cfnTemplateRoot } = require('../provider-utils/awscloudformation');
```

Comment:
Why the curly braces?

Reply:
It's convention to destructure imports like this instead of `const cfnTemplateRoot = require('../provider-utils/awscloudformation').cfnTemplateRoot`